projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -2140,95 +2140,6 @@ public function get_dismissed_notices() {
 	public static function can_manage_connection( $connection_data ) {
 		return current_user_can( 'edit_others_posts' ) || get_current_user_id() === (int) $connection_data['user_id'];
 	}
-
-	/**
-	 * Display a Fediverse actor Open Graph tag when the post author has a Mastodon connection.
-	 *
-	 * @see https://blog.joinmastodon.org/2024/07/highlighting-journalism-on-mastodon/
-	 *
-	 * @param array $tags Current tags.
-	 *
-	 * @return array
-	 */
-	public function add_fediverse_creator_open_graph_tag( $tags ) {
```

Comment:
I'm opting to delete those methods without deprecating them since they were never used in the wild. The hooks that related to those 2 methods were deleted in #38612.